tests/casperjs/util/include/util.js

```diff
@@ -68,14 +68,10 @@ var mainUtil = (function() {
      * @private
      */
     var generateRandomString = function(toLowerCase) {
-        if (toLowerCase && !_.isBoolean(toLowerCase)) {
-            throw new Error('An invalid value for toLowerCase has been provided');
-        }
-
-        // Generate a random string
-        var rndString = _.sample('ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789', 10).join('');
-        if (toLowerCase) {
-            rndString = rndString.toLowerCase();
+        var rndString = '';
```

Comment:
I needed to revert this as underscore or any other dependencies can't be injected here as we're not in a DOM environment.